src/main/kotlin/stringcalculator/Number.kt

```diff
@@ -0,0 +1,11 @@
+package stringcalculator
+
+class Number(valueString: String) {
```

Comment:
아래와 같은 주 생성자(primary constructor)를 만든 후에 자기 생성자 호출하는 `this()`를 사용하면 어떨까요?

```suggestion
class Number(val value: Int) {
```